### ISSUE TITLE
[MSFT-82] [Chrome] The NVDA doesn't read item name when user try to mouse hover on them

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2793]
**Steps to Reproduce**
1. Navigate to https://accounts.topcoder.com/member/registration
2. Try to mouse hover on any text-box or "GitHub/ Google/ Facbook" icons

**Expected Result**
The NVDA should read item name when user try to mouse hover on them

**Actual Result**
The NVDA doesn't read item name when user try to mouse hover on them

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.2.3 Audio Description or Media Alternative (Prerecorded)
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: 

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F12

**Comment by mishacucicea** [comment]
seems duplicate of #2834

**Comment by Oanh-and-only-Oanh** [comment]
closing because it's a duplicate bug
